GetOldTweets3/manager/ConcurrentTweetManager.py

```diff
@@ -0,0 +1,78 @@
+# Author: Giulio Neusch-Frediani - www.github.com/giulionf
+from datetime import datetime
+from threading import Thread
+from queue import Queue
+from queue import Empty
+import copy
+
+from GetOldTweets3.manager.TweetManager import TweetManager
+
+DATE_FORMAT = "%Y-%m-%d"
+FIRST_TWEET_DATE = datetime.strptime("2006-03-21", DATE_FORMAT)
+
+
+class ConcurrentTweetManager:
+
+    @staticmethod
+    def getTweets(tweetCriteria, receiveBuffer=None, bufferLength=100, proxy=None, debug=False, worker_count=1, ordered=True):
+        # Init the queues
+        time_spans = Queue()
+        tweets = []
+        workers = []
+
+        # Split the date in smaller parts
+        since_date = datetime.strptime(tweetCriteria.since, DATE_FORMAT) if tweetCriteria.since else FIRST_TWEET_DATE
+        until_date = datetime.strptime(tweetCriteria.until, DATE_FORMAT) if tweetCriteria.until else datetime.now()
+        date_diff_per_worker = (until_date - since_date) / worker_count
+
+        # Create a TweetCriteria that can be cloned as Model by the Workers
+        criteria = copy.deepcopy(tweetCriteria)
+        criteria.setMaxTweets(criteria.maxTweets / worker_count)
```

Comment:
This is obviously an error. We can't take first `criteria.maxTweets / worker_count` tweets from each thread...

Reply:
If we do not split the maxTweets by the worker count, each worker will pull maxTweet tweets. Say maxtweet is 200 and workers = 2 than each worker should pull 100. 
Problem is that if we set maxTweet to 1000 we will not get the first 1000 tweets. But there is really no other solution since we don‘t know how many tweets there are in total...

Reply:
I think we should simply make `--max-tweets` parameter incompatible with parallel downloading. User should choose one or another.

Reply:
Yes we could do that. So then the workers would donwload all tweets! If thats fine with your I will implement it like that!

Reply:
Yes, I think that is the best we can offer